docs/java/java-testing.md

```diff
@@ -21,39 +21,110 @@ The extension supports the following test frameworks:
 - [JUnit 5](https://junit.org/junit5/) (v5.1.0+)
 - [TestNG](https://testng.org/doc/) (v6.8.0+)
 
-> Note: JUnit 3 styled tests are not supported in this extension (for example, `extends junit.framework.TestCase`).
-
-The [Java Test Runner](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-test) works with the [Language Support for Java™ by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java) and [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) extensions to provide the following features:
+The [Test Runner for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-test) works with the [Language Support for Java™ by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java) and [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug) extensions to provide the following features:
 
 - Run/Debug test cases
 - Customize test configurations
 - View test report
 - View tests in Test Explorer
 
-We recommend installing the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which includes the features above, to get the complete Java development experience.
+We recommend installing the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack) to get the complete Java development experience.
 
 <a class="tutorial-install-extension-btn" href="vscode:extension/vscjava.vscode-java-pack">Install the Extension Pack for Java</a>
 
-If you run into any issues when using the features below, you can contact us by entering an [issue](https://github.com/microsoft/vscode-java-pack/issues).
-
 ## Requirements
 
 - JDK (version 11 or later)
 - VS Code (version 1.59.0 or later)
 - [Language Support for Java™ by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java)
 - [Debugger for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-debug)
 
+## Project Setup
+(If you have already setup your Java test framework in your project, you can skip to the [Features](#features) section.)
+
+### JUnit 4
+#### Maven
+Add following configuration into your `pom.xml`:
+
+```xml
+<dependency>
+  <groupId>junit</groupId>
+  <artifactId>junit</artifactId>
+  <version>(YOUR_JUNIT_VERSION)</version>
+  <scope>test</scope>
+</dependency>
+```
+
+#### Gradle
+Make sure following lines are added in your `build.gradle`:
+
+```groovy
+plugins {
+    java
+}
+
+dependencies {
+    testImplementation('junit:junit:(YOUR_JUNIT_VERSION)')
+}
+```
+
+#### Unmanaged Folder
+If your project does not use any build tools. You need to download the following JARs and add them to the project classpath (via setting `java.project.referencedLibraries`):
```

Comment:
should we include a screenshot or highlight the button to add a dependency? I feel like this is the area where we are getting a lot of feedback

Reply:
How about link to this section: https://code.visualstudio.com/docs/java/java-project#_library-configuration. And we can attach screenshot there.

Reply:
sounds good